### PR TITLE
Update microsoft-365-groups-expiration-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
Not sure if this can be added as another line but Added a note about Yammer native mode.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4263

File: microsoft-365/solutions/microsoft-365-groups-expiration-policy.md [hunk 1]
```diff
@@ -33,6 +33,11 @@ Groups that are actively in use are renewed automatically. Any of the following
 - Outlook - join group, read or write group message from the group, and like a message (Outlook on the web).
 - Teams - visiting a teams channel.
 
+
+> [!Note]
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/microsoft-365-groups-expiration-policy.md]:
```suggestion
> [!NOTE]
```

File: microsoft-365/solutions/microsoft-365-groups-expiration-policy.md [hunk 2]
```diff
@@ -33,6 +33,11 @@ Groups that are actively in use are renewed automatically. Any of the following
...
+> Microsoft 365 Groups converted by Yammer in native mode do not renew automatically and the yammer activities donot count for automatic renewal. For more information about Native mode, See[Overview of Native Mode](https://docs.microsoft.com/en-us/yammer/configure-your-yammer-network/overview-native-mode)
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/microsoft-365-groups-expiration-policy.md]:
```suggestion
> Microsoft 365 Groups converted by Yammer in Native Mode do not renew automatically and the Yammer activities do not count towards automatic renewal. For more information about Native Mode, see [Overview of Native Mode](https://docs.microsoft.com/yammer/configure-your-yammer-network/overview-native-mode).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by MikePlumleyMSFT** [comment]
@ArunMSFT can you review?

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by Josegto1 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
Gentle reminder @MikePlumleyMSFT Please merge this PR. Thanks.

**Comment by MikePlumleyMSFT** [comment]
@ArunMSFT can you review?

**Pull request merged**